cristin-import/src/test/java/no/unit/nva/cristin/lambda/CristinEntryEventConsumerTest.java

```diff
@@ -287,8 +287,22 @@ public void handlerThrowsInvalidIsbnRuntimeExceptionWhenTheIsbnIsInvalid() throw
     }
 
     @Test
-    public void handlerThrowsInvalidIssnRuntimeExceptionWhenTheIssnIsInvalid() throws JsonProcessingException {
-        JsonNode cristinObjectWithInvalidIssn = CristinDataGenerator.objectWithInvalidIssn();
+    public void handlerThrowsInvalidIssnRuntimeExceptionWhenTheBookIssnIsInvalid() throws JsonProcessingException {
+        JsonNode cristinObjectWithInvalidIssn = CristinDataGenerator.bookObjectWithInvalidIssn();
+        AwsEventBridgeEvent<FileContentsEvent<JsonNode>> awsEvent =
+                CristinDataGenerator.toAwsEvent(cristinObjectWithInvalidIssn);
+        InputStream inputStream = IoUtils.stringToStream(awsEvent.toJsonString());
+
+        Executable action = () -> handler.handleRequest(inputStream, outputStream, CONTEXT);
+
+        RuntimeException exception = assertThrows(RuntimeException.class, action);
+        Throwable cause = exception.getCause();
+        assertThat(cause, is(instanceOf(InvalidIssnRuntimeException.class)));
```

Comment:
The InvalidIssnRutimeException is a RuntimeException and should be thrown directly, not being the cause.